Update global.R

ltcarendt opened the pull request:
Previous version did not iterate over all items in IHME zip file; it only looked at first and last items.
New version *correctly* iterates over all items in IHME zip file to find desired CSV file name

ltcarendt committed:
Update global.R

Previous version did not iterate over all items in IHME zip file; it only looked at first and last.
New version *correctly* iterates over all items in IHME zip file to find desired CSV file name

galarcon0308 reviewed (changes requested):
Yeah I came across an issue and had to revert it back. They also changed their column position (annoying). Anyway we can use the same method you have and look for just a csv? They have change the csv file name before along with the going back and forth between a capital "H". The only consistent thing so far is that it always has two files regardless of their name. A .csv and a .pdf

I think if we search just based on the extension that might be the best for generality? Let me know your thoughts and if this is easily doable in the code you just provided. I think its as easy as getting rid of the first grepl condition

galarcon0308 commented:
Let me know if you can read the comment or not, referring to changing to searching for just a csv since they have sometimes changed the file name also

ltcarendt committed:
Update global.R

1. Infer CSV filename from IHME zip file by searching zip file list for ".csv"
2. Infer IHME location variable name by searching column names for "loc"

ltcarendt committed:
Update global.R

Using https://covid-19.bsvgateway.org/forecast/forecast_metadata.json to find "most_recent_date" for LANL data

ltcarendt reviewed (commented):
May want to add 
~~library(jsonlite)~~
to initialization
edit: it's already there!

galarcon0308 reviewed (commented):
Awesome job!
